axolotl-web/src/router/router.js

```diff
@@ -9,75 +9,77 @@ export default new Router({
-    name: "chat",
-    props: route => ({
-      chatId: route.params.id,
-    }),
-    component:  () => import("@/pages/MessageList.vue")
-  },
-  {
-    path: "/",
-    name: "register",
-    component:  () => import("@/pages/Register.vue")
-  },
-  {
-    path: "/verify",
-    name: "verify",
-    component:  () => import("@/pages/Verification.vue")
-  },
-  {
-    path: "/password",
-    name: "password",
-    component:  () => import("@/pages/Password.vue")
-  },
-  {
-    path: "/setPassword",
-    name: "setPassword",
-    component:  () => import("@/pages/SetPassword.vue")
-  },
-  {
-    path: "/contacts",
-    name: "contacts",
-    component:  () => import("@/pages/Contacts.vue")
-  },
-  {
-    path: "/settings",
-    name: "settings",
-    component:  () => import("@/pages/Settings.vue")
-  },
-  {
-    path: "/about",
-    name: "about",
-    component:  () => import("@/pages/About.vue")
-  },
-  {
-    path: "/devices",
-    name: "devices",
-    component:  () => import("@/pages/DeviceList.vue")
-  },
-  {
-    path: "/newGroup",
-    name: "newGroup",
-    component:  () => import("@/pages/NewGroup.vue")
-  },
-  {
-    path: "/editGroup/:id",
-    name: "editGroup",
-    component:  () => import("@/pages/EditGroup.vue")
-  },
-  { path: '/a', redirect: () => {
-      // { path: '/a', redirect: to => {
+    {
+      path: "/chatList",
+      name: "chatList",
+      component: () => import("@/pages/ChatList.vue")
+    },
+    {
+      path: "/chat/:id",
+      name: "chat",
+      props: route => ({
+        chatId: route.params.id,
+      }),
+      component: () => import("@/pages/MessageList.vue")
+    },
+    {
+      path: "/",
+      name: "register",
+      component: () => import("@/pages/Register.vue")
+    },
+    {
+      path: "/verify",
+      name: "verify",
+      component: () => import("@/pages/Verification.vue")
+    },
+    {
+      path: "/password",
+      name: "password",
+      component: () => import("@/pages/Password.vue")
+    },
+    {
+      path: "/setPassword",
+      name: "setPassword",
+      component: () => import("@/pages/SetPassword.vue")
+    },
+    {
+      path: "/contacts",
+      name: "contacts",
+      component: () => import("@/pages/Contacts.vue")
+    },
+    {
+      path: "/settings",
+      name: "settings",
+      component: () => import("@/pages/Settings.vue")
+    },
+    {
+      path: "/about",
+      name: "about",
+      component: () => import("@/pages/About.vue")
+    },
+    {
+      path: "/devices",
+      name: "devices",
+      component: () => import("@/pages/DeviceList.vue")
+    },
+    {
+      path: "/newGroup",
+      name: "newGroup",
+      component: () => import("@/pages/NewGroup.vue")
+    },
+    {
+      path: "/editGroup/:id",
+      name: "editGroup",
+      component: () => import("@/pages/EditGroup.vue")
+    },
+    {
+      path: '/a', redirect: () => {
+        // { path: '/a', redirect: to => {
         // const { hash, params, query } = to
```

Comment:
I see nothing more than whitespace changes here... How do that solve the error?
Also, maybe drop those comments?

Reply:
There was the idention of the file wrong, i had to add things there for debugging and then i removed them again.